recipes/supersmoother/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "supersmoother" %}
+{% set version = "0.4" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "aeef4e1b00c32316d624ea7e3ac87c244bf2e59abbb6c042e7791f69ae0669cb"
+
+build:
+  number: 0
```

Comment:
I think you could make this a `noarch` package, which will mean less CI usage once it is merged. See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python